ui/src/hosts/containers/HostPage.js

```diff
@@ -168,21 +168,16 @@ class HostPage extends Component {
     const {
```

Comment:
oooh wouldn't it be sooo nice if this page were TS??  :)

Reply:
heh soooo tempting...